docs/en/setup/advanced/MeterReporter.md

```diff
@@ -0,0 +1,110 @@
+# Python Agent Meter Reporter
+
+To enable or disable this feature, you will need to set some environment variables.
+
+
+## Enabling the feature (default)
+```Python 
+os.environ['SW_AGENT_METER_REPORTER_ACTIVE'] = 'True'
+``` 
+or
+```bash
+export SW_AGENT_METER_REPORTER_ACTIVE=True
+```
+
+## Disable the feature
+```Python 
+os.environ['SW_AGENT_METER_REPORTER_ACTIVE'] = 'False'
+``` 
+or
+```bash
+export SW_AGENT_METER_REPORTER_ACTIVE=False
+```
+## Counter
+* `Counter` API represents a single monotonically increasing counter, automatic collect data and report to backend.
+```python
+tg_ls = [MeterTag("key", "value")]
+c = Counter('c2', CounterMode.INCREMENT, tg_ls)
+# or with more compact way
+# Counter('c2', CounterMode.INCREMENT).tag('key1', 'value1').tag('key2', 'value2')
+c.build()
+c.increment(2)
+```
+### Syntactic sugars
+```python
+c = Counter('c2', CounterMode.INCREMENT)
+c.build()
```

Comment:
Looks like every meter should invoke `build` after construction, what about just embed the `build` inside the construction of the meters? So that users won't forget to call the `build` method?

Reply:
You are right. This is the sequelae of imitating java agent.

Reply:
The reason we need build method because we separate tags definition from construction.

Reply:
> The reason we need build method because we separate tags definition from construction.

Thanks for reminding. Here also separate tags definition from construction, so seems `build` is still needed.